app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/injection/module/OutputActivityModule.java

```diff
@@ -1,7 +1,16 @@
 package com.benoitquenaudon.daggerbundleinjectiondemo.injection.module;
 
+import com.benoitquenaudon.daggerbundleinjectiondemo.BundleService;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.SomeInput;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.scope.ScopeActivity;
 import com.benoitquenaudon.daggerbundleinjectiondemo.ui.output.OutputActivity;
 import dagger.Module;
+import dagger.Provides;
 
-@Module abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+@Module public abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+  public static final String EXTRA_SOME_INPUT = "com.benoitquenaudon.demo.SOME_INPUT";
```

Comment:
That is a valid concern. My only thinking was that having info for something specific on a module that's more general. But you're right, they should at least be somewhere, that will be for all of them.

Considering they are accessed by more than just the provides method, they don't have to be tied to it.